lib/resources/r4/organization.yaml

```diff
@@ -0,0 +1,127 @@
+            <li> Must contain only an NPI, OID or TAXID system. </li>
+          </ul>
+        </li>
+      </ul>
+    children:
+
+      - name: type
+        required: 'No'
+        cardinality: 0..1
+        type: CodeableConcept
+        description: A coded type for the identifier that can be used to determine which identifier to use for a specific purpose.
+        url: http://hl7.org/fhir/r4/datatypes-definitions.html#Identifier.type
+        binding:
+          description: A coded type for the identifier that can be used to determine which identifier to use for a specific purpose.
+          terminology:
+            - display: Identifier Type Codes
+              system: http://terminology.hl7.org/CodeSystem/v2-0203
+              info_link: http://hl7.org/fhir/r4/valueset-identifier-type.html
+            - display: Millennium Organization Identifier (Alias) Types
+              system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/334
+              info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-334-organization-alias-type
+
+  - name: type
+    required: 'No'
+    cardinality: 0..*
+    type: CodeableConcept
+    description: Kind of organization.
+    url: https://hl7.org/fhir/R4/organization-definitions.html#Organization.type
+    action: create
+    example: |
+      {
+        "type": [
+          {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/organization-type",
+                "code": "govt"
+              }
+            ]
+          }
+        ]
+      }
+
+    binding:
+      description: Indicates the type of Organization.
+      terminology:
+        - display: OrganizationType
+          system: http://terminology.hl7.org/CodeSystem/organization-type
+          info_link: http://hl7.org/fhir/r4/valueset-organization-type.html
+        - display: Millennium Organization Identifier (Alias) Types
+          system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/278
+          info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-278-organization-type
+
+  - name: resourceType
+    required: 'Yes'
+    cardinality: 1..1
+    type: string
+    description: The type of the FHIR resource.
+    action: create
+    example: |
+      {
+        "resourceType": "Organization"
+      }
+    note: resourceType must be Organization.
+    url: https://hl7.org/fhir/R4/json.html#resources
+
+  - name: active
+    required: 'No'
+    cardinality: 0..1
+    type: boolean
+    description: Whether the organization's record is still in active use.
+    action: create
+    example: |
+      {
+        "active": true
+      }
+    note: If set, value must be true.
+    url: https://hl7.org/fhir/R4/organization-definitions.html#Organization.active
+
+  - name: name
+    required: 'Yes'
+    cardinality: 0..1
+    type: OrganizationName
```

Comment:
this is a string

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/326/commits/884311d2afad568b4d627e742466974d702778a3